R/LinReg.R

```diff
@@ -1,6 +1,6 @@
 LinReg<- function(model, particles=1000, plot=FALSE) {
 
-    data(radiata)
+    data(radiata, envir = environment())
```

Comment:
Is this really the best we can do / should do?  I haven't looked at this (and the manual) in some time but I believe I have dealt with this differently in the past.